tests/dsp/dsp_modes/README.rst

```diff
@@ -0,0 +1,56 @@
+DSP-style block with different modes
+++++++++++++++++++++++++++++++++++++
+
+The possible modes are
+
+* Combinational only (i.e. |dsp_combinational|_)
```

Comment:
For some reason this item in the list isn't rendering correctly?

Reply:
Fixed that. Should be rendering correctly now.